src/System.IO.FileSystem/src/System/IO/Win32FileSystemEnumerable.cs

```diff
@@ -492,13 +496,13 @@ private static String GetNormalizedSearchCriteria(String fullSearchString, Strin
             return searchCriteria;
         }
 
-        private static String GetFullSearchString(String fullPath, String searchPattern)
+        private static string GetFullSearchString(string fullPath, string searchPattern)
         {
             Contract.Requires(fullPath != null);
             Contract.Requires(searchPattern != null);
 
             PathHelpers.ThrowIfEmptyOrRootedPath(searchPattern);
-            String tempStr = Path.Combine(fullPath, searchPattern);
+            string tempStr = Path.Combine(fullPath, searchPattern);
```

Comment:
I'm assuming you're correct, but I didn't have a chance to investigate myself. If I don't have a chance to get to it for this PR, I'll open an up-for-grabs issue to track.